src/content/platform-overview/cluster-management/products/_index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Our products
+description: Overview of Giant Swarm cluster management products.
+last_review_date: 2023-06-20
+weight: 10
+aliases:
+menu:
+  main:
+    parent: platform-overview-cluster-management
+    identifier: cluster-management-products
+user_questions:
+  - Which different products does Giant Swarm offer?
+  - Which cloud providers are supported?
+owner:
+  - https://github.com/orgs/giantswarm/teams/area-kaas
+---
+
+## Product families
+
+We have two _product families_: _Cluster API (CAPI)_ based and _vintage_ products.
```

Comment:
I would rather see this in this structure:

We support different providers: AWS, Azure, Vmware, GCP

With the move to the open standard of CAPI we've done a lot of changes to the cluster management and it is in different states on different providers. Cluster managed by our old (pre-CAPI) operators are called vintage. All new development happens now on CAPI clusters. We also have an old KVM/Baremetal/Vmware stack that is legacy by now and finally phased out this summer.

The majority of our customers are still running on AWS vintage but CAPA (CAPI for AWS) is already GA. One main benefit is that we've implemented private clusters with private dns, private acme challenge, egress proxies and transit gateways. EKS support is under development.

CAPZ (CAPI for Azure) is currently in beta and for new customers we wouldn't recommend to use the vintage version anymore.  All other providers are based on CAPI anyway.

We will help you to pick the right version and also help all existing customers to migrate between the generations of our product. In most cases those processes are fully automated and can be tested in advance.

WDYT? /cc @puja108

Reply:
yeah, I do get the thoughts behind "families", but especially as we also want to offer migration paths and move everyone over, it might make more sense to talk about the evolution within providers here, i.e. AWS (with our own controllers) towards CAPA being more upstream, brining more flexiblity, features, pricavy, EKS,...

Reply:
I reworded it a lot now, and use the term "generations" instead. Please have a look – it makes a lot more sense now, I think.

Let's include some benefits of _CAPI-based_ over _vintage_ – do you have a suggestion? I'd say 1) less vendor lock-in i.e. more open-source operators (more knowledge in engineering talents), 2) more flexibility e.g. for networking (link to private cluster support), 3) development of integration with cloud provider's KaaS / control planes (EKS, AKS, ...).